app/controllers/users_controller.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    @users = User.all
+    render json: @users
+  end
+
+  def show
+    @user = User.find(params[:id])
```

Comment:
If the user doesn't exist with this `id` will server return `404 not found error` in json?

Reply:
yes it will